packages/nodes/table/src/queries/getTableRowIndex.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  findNodePath,
+  TElement,
+  TReactEditor,
+  Value,
+} from '@udecode/plate-common';
+
+/**
+ * Get table row index of a cell node.
+ */
+export const getTableRowIndex = <V extends Value>(
+  editor: TReactEditor<V>,
+  cellNode: TElement
+) => {
+  const path = findNodePath(editor, cellNode);
+  if (!path) return 0;
+  return path[path.length - 2];
```

Comment:
can use `Path.parent`

Reply:
I'm not sure how `Path.parent` would help here. We want something that maps `[a, b, c, d] -> c`; `parent` takes a path and returns a node.